azurelinuxagent/common/utils/shellutil.py

```diff
@@ -110,6 +110,34 @@ def run_get_output(cmd, chk_err=True, log_cmd=True, expected_errors=[]):
     return 0, output
 
 
+def _encode_command_output(output):
+    return ustr(output, encoding='utf-8', errors="backslashreplace")
+
+
+def run_command(command):
```

Comment:
let's add a comment on run_get_output to explain when/why to use this new function instead

Reply:
Done.